src/ucm/event/event.c

```diff
@@ -26,6 +26,15 @@
 #include <string.h>
 #include <errno.h>
 
+
+#define UCM_NATIVE_EVENT_VM_MAPPED (UCM_EVENT_MMAP  | UCM_EVENT_MREMAP | \
+                                    UCM_EVENT_SHMAT | UCM_EVENT_SBRK)
+
+#define UCM_NATIVE_EVENT_VM_UNMAPPED (UCM_EVENT_MMAP   | UCM_EVENT_MUNMAP | \
```

Comment:
minor: you could use `UCM_NATIVE_EVENT_VM_MAPPED ` here, because UNMAPPED set is an extension of MAPPED